internal/providersdkv2/resource_vault_cluster_config_test.go

```diff
@@ -69,6 +69,51 @@ func TestGetValidObservabilityConfig(t *testing.T) {
 			},
 			expectedError: "newrelic configuration is invalid: configuration information missing",
 		},
+		"http missing params": {
```

Comment:
These tests are the only difference between this PR and #660, is that right?

Reply:
Yes